descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "patron",
-      "version": "3.0",
+      "version": "3.1",
```

Comment:
Given that endpoints are being removed, I would interpret that as a breaking compatibility change. Is that not the case here?

Reply:
That is a fair point. I was on the fence mentioning this in my review since I don't think there is any other client for this module expect edge-patron and I don't believe we use those deleted endpoints. But that shouldn't keep us from doing the right thing. Thanks for keeping us honest Marc.
```suggestion
      "version": "4.0",
```

Reply:
done